x-pack/solutions/security/plugins/security_solution/public/management/pages/event_filters/view/components/form.test.tsx

```diff
@@ -28,6 +28,8 @@ import {
 import type { IHttpFetchError } from '@kbn/core-http-browser';
 import { buildPerPolicyTag } from '../../../../../../common/endpoint/service/artifacts/utils';
 
+jest.setTimeout(15_000); // Costly tests, hitting 2 seconds execution time locally
```

Comment:
Ran locally 50 times with no failures. As commented, some tests in this file hit 2 seconds execution time, it is likely that the default 5 seconds is not enough for it to complete in CI. Hence changing the default timeout.